website/pages/style.css

```diff
@@ -82,6 +82,237 @@
   color: white;
 }
 
+/* Print Styles - Hide Elements */
+@media print {
+  iframe,
+  .g-mega-nav,
+  .g-product-subnav,
+  .g-footer,
+  [aria-hidden='true'],
+  [id='sidebar'],
+  [id='edit-this-page'],
+  [id='jump-to-section'],
+  [id='__next-build-watcher'] {
+    display: none !important;
+  }
+}
```

Comment:
Currently the global scope here headers disappear from the home page as well when printing:

![Screen Shot 2020-02-19 at 16 56 03](https://user-images.githubusercontent.com/4624598/74880590-881c0380-5339-11ea-92fa-6d81687b679e.png)

This is probably fine, but feels a bit like the work here is leaking into other pages too. What do we think of putting all `@media print` styles under `#p-docs` for now, since that's the only place we're looking at really intentionally at the moment?

On a related note, since there are only a couple of `@media screen` blocks, maybe it could even make sense to put everything in this PR under a single `print.css` file, eg in `website/components/docs-page/print.css`.

Reply:
Rather than add `#p-docs`, I think it would be easier to maintain the print styles if their selectors are written to match their equivalents (usually _dot g_ components).

I have moved the styles into their own file.

Reply:
I think this makes sense, but am still a bit wary that `website/pages/print.css` as opposed to `website/components/docs-page/print.css` implies that these print styles are intended for use on more than just the `docs-page` view (which I don't think is the case, though not 100% sure). Still this is just an opinion on my part, and can see a case that maybe broader use is ultimately what we're aiming for 👍

Reply:
Yea, you are right on that the focus here is the docs page (which is the majority of the website?).

If it helps, tho, this happens to make the `/resources` page just as printable. So in theory other added pages would automatically get these benefits. 😅

Reply:
ah excellent point, i was totally missing that! 😅 the `website/pages/print.css` makes a lot more sense to me now 🙇‍♂️